README.md

```diff
@@ -92,142 +92,101 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Improved the "admin application running" notification checking logic to be less demanding on resources. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed an issue causing many utilities to crash when the GPO to disable data diagnostics was applied.
+
+ - Fixed an issue where updating PowerToys on Windows 11 did not properly update context menu entries, impacting New+, PowerRename, Image Resizer, and File Locksmith.
+ - Updated .NET Packages from 9.0.1 to 9.0.2. Thanks [@snickler](https://github.com/snickler) for this.
+ - Enabled compatibility with VS17.3 and later, for C++23. Thanks [@LNKLEO](https://github.com/LNKLEO) for this.
 
 ### Advanced Paste
 
- - Fixed a crash when the application was exiting. (This was a hotfix for 0.87)
- - Added a Json format validation step to verify if a conversion to Json should be applied.
- - Fixed accessibility issues when using a screen reader.
- - Added support for all BitmapDecoder supported image file types to the Image to Text functionality. Thanks [@daverayment](https://github.com/daverayment)!
- - Fixed an issue causing Advanced Paste initialization errors to hang the PowerToys main process.
+ - Enhanced Advanced Paste by adding media transcoding support, improving UI layouts, refining clipboard handling, and integrating Semantic Kernel for smarter pasting. Thanks for our team and [@snickler](https://github.com/snickler)!
 
 ### FancyZones
 
- - Removed Workspaces Editor from the exclusions list so it can be snapped by FancyZones.
-
-### Keyboard Manager
+ - Fixed accessibility by improving the text for monitors, ensuring clearer naming and help text for screen readers.
 
- - Added an option to make a shortcut remapping only trigger with exact modifiers.
+### Image Resizer
+ - Fixed issues with Width and Height fields in Image Resizer's Custom preset, ensuring empty values no longer cause errors, settings save correctly, and auto-scaling behaves as expected. Thanks [@daverayment](https://github.com/daverayment)!
+ - Fixed accessibility by ensuring screen readers announce selected image dimensions in the combo-box for better navigation.
 
 ### Monaco Preview
 
- - Added support for .resx and .resw files in Peek and File Explorer add-ons. Thanks [@asif4318](https://github.com/asif4318)!
- - Added a setting to make the code minimap toggle-able in Peek and File Explorer add-ons. Thanks [@PesBandi](https://github.com/PesBandi)!
- - Fixed an issue causing Json format preview setting to not be applied correctly.
- - Fixed an issue causing the wrong Monaco assets to be used at runtime.
-
-### Mouse Without Borders
-
- - Fixed an issue causing clipboard to stop working after going through a UAC screen when using the Service mode. Thanks [@YDKK](https://github.com/YDKK)!
+ - Fixed open link in default browser rather than edge. Thanks [@OldUser101](https://github.com/OldUser101)!
```

Comment:
```suggestion
 - Fixed open link in default browser rather than Edge. Thanks [@OldUser101](https://github.com/OldUser101)!
```

Reply:
@cinnamon-msft, i adjusted to "Microsoft Edge" vs just Edge